app/build.gradle.kts

```diff
@@ -193,6 +193,7 @@ android {
     isWarningsAsErrors = true
     isAbortOnError = true
     isCheckReleaseBuilds = false
+    isCheckDependencies = true
```

Comment:
What does this setting do?

Reply:
This ensures that lint is also ran on our modules.